ding2.make

```diff
@@ -160,10 +160,8 @@ projects[sslproxy][version] = "1.0"
 projects[redirect][subdir] = "contrib"
 projects[redirect][version] = "1.0-rc1"
 
-projects[cookiecontrol][subdir] = "contrib"
-projects[cookiecontrol][version] = "1.6"
-# https://drupal.org/node/2174955, fix translatable link.
-projects[cookiecontrol][patch][0] = "http://drupal.org/files/issues/translatable_link_title-2174955-1.patch"
+projects[eu-cookie-compliance][subdir] = "contrib"
+projects[eu-cookie-compliance][version] = "1.14"
```

Comment:
Are you sure? The project name is eu-cookie-compliance, but the filenames are eu_cookie_compliance.* (see also: https://www.drupal.org/node/1603658)